pkgs/applications/networking/instant-messengers/bluejeans/default.nix

```diff
@@ -36,14 +36,15 @@ stdenv.mkDerivation rec {
       --replace-needed libudev.so.0 libudev.so.1 \
       opt/bluejeans/bluejeans-bin
     ln -s $out/opt/bluejeans/bluejeans $out/bin/bluejeans
-    substituteInPlace $out/bin/bluejeans \
-      --replace '#!/bin/bash' '#!${bash}/bin/bash'
     chmod +x $out/bin/bluejeans
+    patchShebangs $out
   '';
 
-  meta = {
+  meta = with stdenv.lib; {
     description = "Video, audio, and web conferencing that works together with the collaboration tools you use every day.";
-    license = stdenv.lib.licenses.unfree;
+    homepage = http://www.bluejeans.com;
```

Comment:
There's a https url, and looks like https://github.com/NixOS/rfcs/pull/45 has gained enough traction that we should start quoting things :)

Reply:
@srhb Oh thanks! This was copypasta from my very old failed attempt to implement this bluejeans package. Also switched to https.